src/transforms/glossary.ts

```diff
@@ -27,11 +55,35 @@ export function addGlossaryDefinition(
 
         if (at == null) return
 
+        const fragment: Descendant[] = Editor.fragment(editor, at)
+        const nodeToInsert: Node[] = []
+        const getToBottom = (descendant: Descendant[]) => {
+            if (Array.isArray(descendant)) {
+                descendant.some((elem: Descendant) => {
```

Comment:
What is this call to `some` doing?

Reply:
I've come to the conclusion that people can have extreme ideas like making a definition with 2 paragraphs. Should I remove it?

Reply:
You answered a different question. I'm asking not why you written this code, but rather what exactly is it doing. As far as I understand, this is just a loop, but if that's the case, why is it implemented using `Array#some` instead of a `for` statement?

Reply:
You're right, I'll change it to for